lib/pages/random_gifs.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:giphy_picker/giphy_picker.dart';
+
+class RandomGifs extends StatefulWidget {
+  @override
+  State<RandomGifs> createState() => _RandomGifsState();
+}
+
+class _RandomGifsState extends State<RandomGifs> {
+  GiphyGif? _gif;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(_gif?.title ?? 'Giphy Picker Demo'),
+      ),
+      body: SafeArea(
+        child: Center(
+          child: _gif == null
+              ? const Text('Pick a gif..')
+              : GiphyImage.original(gif: _gif!),
+        ),
+      ),
+      floatingActionButton: FloatingActionButton(
+        backgroundColor: Colors.blueAccent,
+        child: const Icon(Icons.search),
+        onPressed: () async {
+          final gif = await GiphyPicker.pickGif(
+            context: context,
+            apiKey: 'ENTER YOUR API KEY',
```

Comment:
I suppose there should be api key specified. If that's the case let's make it a compile-time env variable. 
For reference: https://stackoverflow.com/a/61725261/12533631

Reply:
thanks for the reference! i appreciate it

Reply:
with API key and other sensitive data, just ensure they are **never** in your source code/source control system. The simplest thing you can do is to add a separate file, where this var will be defined, and exclude this file from source control (via .gitignore). Then eveyone who clones your repo will have to manually add that file (which is painful, but better than having API key exposed in a public repo)